MonoGame.Framework/Graphics/GraphicsMetrics.cs

```diff
@@ -0,0 +1,65 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework.Graphics
+{
+    /// <summary>
+    /// The rendering information from <see cref="GraphicsDevice"/>. 
+    /// Useful for debugging and profiling purposes. 
+    /// Resetted every frame after draw at <see cref="GraphicsDevice.Present"/>.
+    /// </summary>
```

Comment:
I think you should change the docs here.   Here you should say something like:
```
A snapshot of rendering statistics from `GraphicsDevice.Metrics` to be used for runtime debugging and profiling.
```
The other part...
```
The metrics are reset every frame after draw within <see cref="GraphicsDevice.Present"/>. 
```
Should be moved to the docs for `GraphicsDevice.Metrics` as that is where that information matters.